### ISSUE TITLE
Pagination component should have an input to go to a specific page

### ISSUE THREAD

**Issue by analuizamtg** [vtex/styleguide#815]
**Describe the bug**
In the Collections Admin, the account C&A has more than 1600 pages of Collections. They cannot just go to a specific page, they have to click the next page button until they get where they want.

**Screenshots**
![image](https://user-images.githubusercontent.com/5035494/64184284-a0f33400-ce41-11e9-842d-fa51a82fb60e.png)

**Comment by kevinch** [comment]
Could/should it also have a "go to last page" button?

**Comment by augustocb** [comment]
@analuizamtg Could you describe the scenario of this issue? Do we have a clear task involving a real user or it's a perception of an improvement?

I'm asking you about that because maybe the best way to solve this issue is not by using pagination, but searching/filtering.

**Comment by rsimoens** [comment]
I agree with @augustocb. And even @kevinch suggestion of first/last pages should be done allowing users to choose a column and use it to order itens (up/down).

I believe if an user has to memorize what she/he needs to find it's on page *1345* we have other problems going on, not the pagination itself.

**Comment by cmdalbem** [comment]
@analuizamtg would you want like to rpely to the discussion here or we can consider this closed?

**Comment by ericreis** [comment]
I personally think that we should have two more cases in the pagination:

- First / Last page, as @kevinch mentioned
- Jump 10 (or any other value) next pages (`<<` and `>>`)
